### PR TITLE
Add support for _.initial

### PR DESCRIPTION
**Pull request by salesforcetidbits**
Created a new method to mimic lo's **_.initial(array)**

### CODE REVIEW

File: src/classes/lo_test_initial.cls [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		};
+
+		Test.startTest();
+
+		List<Object> results = lo.initial(testSource);
+
+		Test.stopTest();
+
+		System.assertEquals(9, results.size(), 'Did not get the expected number of results');
+		System.assertNotEquals('j', results[results.size() - 1], 'Unexpected value for last index');
```
Comment (pcon) [inline src/classes/lo_test_initial.cls]: If you could take a look at some of the other classes where we use [expectedResults](https://github.com/apex-lodash/lo/blob/master/src/classes/lo_test_dropRight.cls#L112) and mimic that pattern here.  So that we have a List of  expected strings and compare them.

File: src/classes/lo_test_initial.cls [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+		List<Object> results = lo.initial(testSource);
+
+		Test.stopTest();
+
+		System.assertEquals(0, results.size(), 'Did not get the expected number of results');
+	}
+
+	static testMethod void initial_tenCount_returnsNineItemsList() {
+		String testSource = new List<Object>{
```
Comment (pcon) [inline src/classes/lo_test_initial.cls]: This won't compile since it should be

```java
List<String> testSource = new List<String>{
```

File: src/classes/lo_test_initial.cls [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+			'a', 'b', 'c', 'd', 'e',
```
Comment (pcon) [inline src/classes/lo_test_initial.cls]: Please expand these out to individual lines to meet the code style currently being used.

[Example](https://github.com/apex-lodash/lo/blob/master/src/classes/lo_test_slice.cls#L51)

File: src/classes/lo_test_initial.cls [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+		Test.startTest();
+
+		List<Object> results = lo.initial(testSource);
+
+		Test.stopTest();
+
+		System.assertEquals(0, results.size(), 'Did not get the expected number of results');
+	}
+
```
Comment (pcon) [inline src/classes/lo_test_initial.cls]: Please add an additional `initial_nullCollection_returnsEmptyList` to test that passing in a null list returns the expected results.

**Comment by pcon** [comment]
Thanks for the PR. If you don't mind making those couple of changes I'll gladly merge it in.

File: src/classes/lo_test_initial.cls (hunk 1 above)
Comment (salesforcetidbits) [inline src/classes/lo_test_initial.cls]: Will do, thanks for the quick response!

On Mon, Sep 28, 2015 at 11:20 AM, Patrick Connelly <notifications@github.com
